### PR TITLE
[5.3] Refactoring views to directly call models (com_guidedtours to com_newsfeeds)

### PR DESCRIPTION
**Pull request by Hackwar**
### Summary of Changes
This PR refactors the views to get the model object and then directly call the methods of the model instead of going through the `View::get()` indirection. This refactors the views of the backend components from com_guidedtours to com_newsfeeds.

### Testing Instructions
Everything should work the same.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Review by rdeutz — APPROVED** [review]

File: administrator/components/com_installer/src/View/Installer/HtmlView.php [hunk 1]
```diff
@@ -71,21 +72,21 @@ public function __construct($config = null)
...
-            $showMessage = ($message1 || $message2);
+            $message1          = $state->get('message');
+            $message2          = $state->get('extension_message');
+            $this->showMessage = ($message1 || $message2);
         }
 
-        $this->showMessage = $showMessage;
-        $this->state       = &$state;
-
         $this->addToolbar();
```
Comment (HLeithner) [inline administrator/components/com_installer/src/View/Installer/HtmlView.php]: That change looks a bit strange, can we always expect that the state is a Registry or Object?
If not the reference in line 87 may have a special purpose?

**Review by HLeithner — COMMENTED** [review]
Only the state looks strange the rest seems ok.

File: administrator/components/com_installer/src/View/Installer/HtmlView.php (hunk 1 above)
Comment (Hackwar) [reply administrator/components/com_installer/src/View/Installer/HtmlView.php]: Yes, it always is a Registry object, however you are right, the $state in line 79 is wrong. Fixing it now.

Comment (Hackwar) [reply administrator/components/com_installer/src/View/Installer/HtmlView.php]: Done

Comment (Quy) [reply administrator/components/com_installer/src/View/Installer/HtmlView.php]: See line 84.

Comment (Hackwar) [reply administrator/components/com_installer/src/View/Installer/HtmlView.php]: This has been fixed.

**Comment by laoneo** [comment]
Thanks!

**Comment by wilsonge** [comment]
As an FYI this broke the ability to edit a tour. Was a incredibly simple fix from a bad copy/paste so I've very naughtily fixed it directly https://github.com/joomla/joomla-cms/commit/73c60bca2ad9899d9781d0cb35ef106383d98734 .

**Pull request merged**